### PR TITLE
add description + example for nix flake archive

### PR DESCRIPTION
**Pull request by Mic92**
# Motivation

fixes https://github.com/NixOS/nix/issues/11525

# Context

# Priorities and Process

Add :+1: to [pull requests you find important](https://github.com/NixOS/nix/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc).

The Nix maintainer team uses a [GitHub project board](https://github.com/orgs/NixOS/projects/19) to [schedule and track reviews](https://github.com/NixOS/nix/tree/master/maintainers#project-board-protocol).

### CODE REVIEW

**Comment by Mic92** [comment]
Flaky vm test?

File: src/nix/flake-archive.md [hunk 1]
```diff
@@ -22,8 +22,20 @@ R""(
   # nix flake archive --json --dry-run nixops
   ```
 
+* Upload all flake inputs to a different machine for remote evaluation
+
+  ```
+  # nix flake archive --to ssh://some-machine
+  ```
+
+  On the remote machine the flake can be than access via it's store path. That computed like this:
```
Comment (quatquatt) [inline src/nix/flake-archive.md:31]: Few small typos here, I'd correct this to: 
"On the remote machine the flake can then be accessed via its store path. That's computed like this:"

File: src/nix/flake-archive.md [hunk 2]
```diff
@@ -22,8 +22,20 @@ R""(
...
+
+  ```
+  # nix flake metadata --json | jq -r '.path'
+  ```
+
 # Description
 
-FIXME
+Copy a flake and all its inputs to a store. This useful i.e. to evaluate flakes on a different host.
```
Comment (quatquatt) [inline src/nix/flake-archive.md:39]: Add an "is" between "This" and "useful"

**Review by quatquatt — CHANGES_REQUESTED** [review]

File: src/nix/flake-archive.md (hunk 2 above)
Comment (Mic92) [inline src/nix/flake-archive.md:39]:
```suggestion
Copy a flake and all its inputs to a store. This is useful i.e. to evaluate flakes on a different host.
```

File: src/nix/flake-archive.md (hunk 1 above)
Comment (Mic92) [inline src/nix/flake-archive.md:31]:
```suggestion
  On the remote machine the flake can then be accessed via its store path. That's computed like this:
```

**Review by quatquatt — APPROVED** [review]

**Review by bryanhonof — APPROVED** [review]

**Pull request merged**